src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindIntegrationTest.java

```diff
@@ -2087,6 +2087,7 @@ class Metrics {
     public void checkMetrics() {
       FindIntegrationTest.super.checkMetrics("FindCommand");
       FindIntegrationTest.super.checkDriverMetricsTenantId();
+      FindIntegrationTest.super.checkIndexUsageMetrics("FindCommand", false);
```

Comment:
Since that method is static in base class, is ".super" really needed? Could it just be removed (same for 2 calls above)?